lib/cache.js

```diff
@@ -46,15 +46,17 @@ exports.install = module.exports.install = function(mongoose, options, Aggregate
 				log('save to cache: ', key);
 				cache.set(key, obj);
 			}
-			this.apply(this, arguments);
+      var t = Array.prototype.slice.call(arguments);
```

Comment:
In order to find whether the data is coming from cache or not , I have introduced the third parameter(boolean)